tests/scripts/translate_ciphers.py

```diff
@@ -21,12 +21,541 @@
+             "+DHE-PSK:+AES-128-GCM:+AEAD",
+             None),
+            ("TLS-DHE-PSK-WITH-AES-256-GCM-SHA384",
+             "+DHE-PSK:+AES-256-GCM:+AEAD",
+             None),
+            ("TLS-PSK-WITH-AES-128-CCM",
+             "+PSK:+AES-128-CCM:+AEAD",
+             None),
+            ("TLS-PSK-WITH-AES-256-CCM",
+             "+PSK:+AES-256-CCM:+AEAD",
+             None),
+            ("TLS-DHE-PSK-WITH-AES-128-CCM",
+             "+DHE-PSK:+AES-128-CCM:+AEAD",
+             None),
+            ("TLS-DHE-PSK-WITH-AES-256-CCM",
+             "+DHE-PSK:+AES-256-CCM:+AEAD",
+             None),
+            ("TLS-PSK-WITH-AES-128-CCM-8",
+             "+PSK:+AES-128-CCM-8:+AEAD",
+             None),
+            ("TLS-PSK-WITH-AES-256-CCM-8",
+             "+PSK:+AES-256-CCM-8:+AEAD",
+             None),
+            ("TLS-DHE-PSK-WITH-AES-128-CCM-8",
+             "+DHE-PSK:+AES-128-CCM-8:+AEAD",
+             None),
+            ("TLS-DHE-PSK-WITH-AES-256-CCM-8",
+             "+DHE-PSK:+AES-256-CCM-8:+AEAD",
+             None),
+            ("TLS-RSA-PSK-WITH-CAMELLIA-128-GCM-SHA256",
+             "+RSA-PSK:+CAMELLIA-128-GCM:+AEAD",
+             None),
+            ("TLS-RSA-PSK-WITH-CAMELLIA-256-GCM-SHA384",
+             "+RSA-PSK:+CAMELLIA-256-GCM:+AEAD",
+             None),
+            ("TLS-PSK-WITH-CAMELLIA-128-GCM-SHA256",
+             "+PSK:+CAMELLIA-128-GCM:+AEAD",
+             None),
+            ("TLS-PSK-WITH-CAMELLIA-256-GCM-SHA384",
+             "+PSK:+CAMELLIA-256-GCM:+AEAD",
+             None),
+            ("TLS-DHE-PSK-WITH-CAMELLIA-128-GCM-SHA256",
+             "+DHE-PSK:+CAMELLIA-128-GCM:+AEAD",
+             None),
+            ("TLS-DHE-PSK-WITH-CAMELLIA-256-GCM-SHA384",
+             "+DHE-PSK:+CAMELLIA-256-GCM:+AEAD",
+             None),
+            ("TLS-RSA-PSK-WITH-AES-256-GCM-SHA384",
+             "+RSA-PSK:+AES-256-GCM:+AEAD",
+             None),
+            ("TLS-RSA-PSK-WITH-AES-128-GCM-SHA256",
+             "+RSA-PSK:+AES-128-GCM:+AEAD",
+             None),
+        ]
+
+        for m, g_exp, o_exp in ciphers:
+
+            if g_exp is not None:
+                g = translate_gnutls(m)
+                self.assertEqual(g, g_exp)
+
+            if o_exp is not None:
+                o = translate_ossl(m)
+                self.assertEqual(o, o_exp)
+
+    def test_cipher_format(self):
+        """
+        Ensure translate_ciphers.py can take names in the expected
+        format and return them in the format compat.sh will expect.
+        """
+        # Ciphers in Mbed TLS format
+        ciphers = "TLS-ECDHE-ECDSA-WITH-NULL-SHA                  \
+                   TLS-ECDHE-ECDSA-WITH-3DES-EDE-CBC-SHA          \
+                   TLS-ECDHE-ECDSA-WITH-AES-128-CBC-SHA           \
+                   TLS-ECDHE-ECDSA-WITH-AES-256-CBC-SHA           \
+                  "
+        ciphers = "%s                                             \
```

Comment:
Hmm, my first thought was `ciphers += "..."` looks simpler. But on second thought, why not just make it a single string literal? And while at it, why not use `"""` rather than the tedious `\` for line continuation?

Reply:
Originally it was done to mimic the bash script `test_translate_ciphers_format.sh`, which in turn was used to mimic `compat.sh`. After converting it to python it became redundant as it was not really testing anything. It has been removed I believe.

Reply:
Indeed, for some reason I missed the next commit. Sorry for the noise.